test/ressources/npmignore/.npmignore

```diff
@@ -0,0 +1 @@
+ignore.txt
```

Comment:
The name of the directory should be renamed from `ressources` to `fixtures`.

Reply:
Done